tests/spread/integration/apache2/task.yaml

```diff
@@ -0,0 +1,23 @@
+summary: Integration tests for apache2
+
+execute: |
+  rootfs="$(install-slices apache2_bins base-passwd_data util-linux_cli-helpers)"
+
+  mkdir -p "${rootfs}/dev"
+  mount --rbind /dev "${rootfs}/dev"
+  echo "ServerName localhost" >> "${rootfs}/etc/apache2/apache2.conf"
+  test "$(chroot "${rootfs}" a2query -M)" = "event"
+  test "$(chroot "${rootfs}" a2dismod mpm_event | head -n 1)" = "Module mpm_event disabled."
+  test "$(chroot "${rootfs}" a2enmod mpm_prefork | grep -o "Enabling module mpm_prefork." )" = \
+  "Enabling module mpm_prefork."
+  test "$(chroot "${rootfs}" a2dissite 000-default | head -n 1)" = "Site 000-default disabled."
+  test "$(chroot "${rootfs}" a2ensite 000-default | head -n 1)" = "Enabling site 000-default."
+  test "$(chroot "${rootfs}" a2disconf charset | head -n 1)" = "Conf charset disabled."
+  test "$(chroot "${rootfs}" a2enconf charset | head -n 1)" = "Enabling conf charset."
+  mkdir -p "${rootfs}/var/www/html"
+  echo "Hello, Apache is working!" > "${rootfs}/var/www/html/index.html"
```

Comment:
Do we need to overwrite the default index.html?
Or can we just leave the default one in place?